### ISSUE TITLE
Error: Connection reset by peer

### ISSUE THREAD

**Issue by latelylk** [ethankrein/Trickcord-Treat-Sniper#10]
Usually isn't an issue, but occasionally a user can experience connection dropouts. All that needs to be done is the error needs to be handled.
```
events.js:291
      throw er; // Unhandled 'error' event
      ^

Error: Connection reset by peer
```

**Comment by zefrus1330** [comment]
there is error 
Error: Cannot find module 'eris'
Require stack:

**Comment by latelylk** [comment]
Fixed as of 5e17640
